server/models/reporttype.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  const ReportType = sequelize.define('ReportType', {
+    id: {
+      type: DataTypes.UUID,
+      primaryKey: true,
+      defaultValue: DataTypes.UUIDV4,
+      unique: true,
+      allowNull: false,
+    },
+    name: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    }
+  }, {});
+  ReportType.associate = function (models) { };
+  return ReportType;
+};
```

Comment:
Good job all round. Do you plan to add associations to the model later on?

Reply:
@TheDrizzyWay Thanks. Yes, I actually added associations to `Report` in my `report article feature`. A `reportType` has many `reports`